apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -101,4 +165,9 @@ if (!('BridgeServiceMixin' in self)) {
       { methods: METHODS, streams: STREAMS, events: EVENTS }
     )
   );
+
+  // Automatically init service if it's run inside worker.
+  if ('importScripts' in self) {
```

Comment:
I thought the same while using the bridge mixin, but maybe checking the `importScripts` is fairly enough since it doesn't really need to depend on worker context except this case.